src/parser/xmlHandler.js

```diff
@@ -118,7 +118,8 @@ class XMLHandler {
   mapObject(node, nsContext, descriptor, val) {
     if (val == null) return node;
     if (typeof val !== 'object' || (val instanceof Date)) {
-      node.text(val);
+      parseXml(node, val, null);
```

Comment:
I'm not sure if this logic is correct unless parseXml can deal with non-xml string or date.